### ISSUE TITLE
Pre check Cloudflare and OpenDNS

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/Blocking-Middleware#104]
This is non urgent, but may be needed:

1. We receive some OpenDNS unblock requests which are out of scope.
2. We could improve this by reminding users which categories we are blocking, and that their site will not be unblocked if it legitimately belongs to those categories. They should only report if it should not belong to those categories.
3. We could require an admin / volunteer to approve those requests after a quick sanity check

I don't propose we do this yet however, but keep an eye in case it becomes a significant volume of complaints.
